api/fluentbitoperator/v1alpha2/plugins/output/firehose_types.go

```diff
@@ -0,0 +1,79 @@
+package output
+
+import (
+	"strconv"
+
+	"kubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2/plugins"
+	"kubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2/plugins/params"
+)
+
+// +kubebuilder:object:generate:=true
+
+// The Firehose output plugin, allows to ingest your records into AWS Firehose.
+// https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit
+type Firehose struct {
```

Comment:
Variable naming needs to be adjusted. First, it is not recommended to use `_ `in the definition of crd, for example, `delivery_stream` should be changed to `deliveryStream`. Secondly, for acronyms, use uppercase, such as `Arn => ARN`, `Sts => STS`.

Reply:
Great - I'll fix!